src/backend/chess/app/chess.hoon

```diff
@@ -76,570 +61,946 @@
-                      %agent  [who.action %chess]
-                      %watch  /game/(scot %da game-id)/moves
-                  ==
-                  ::  add our new game to the list of active games
-                  :*  %give  %fact  ~[/active-games]
-                      %chess-game  !>(new-game)
-                  ==
-                  ::  tell our frontend we accepted the challenge
-                  :*  %give  %fact   ~[/challenges]
-                      %chess-update  !>([%challenge-replied who.action])
-                  ==
-              ==
-          %=  this
-            ::  remove our challenger from challenges-received
-            challenges-received  (~(del by challenges-received) who.action)
-            ::  if the poke came from our ship, delete
-            ::  the challenge from our `challenges-sent`
-            challenges-sent  ?:  =(who.action our.bowl)
-                               (~(del by challenges-sent) our.bowl)
-                             challenges-sent
-            ::  put our new game into the map of games
-            games  (~(put by games) game-id [new-game *chess-position *(map @t @ud) | | | | | | |])
-          ==
-        %decline-game
-          =/  challenge  (~(get by challenges-received) who.action)
-          ?~  challenge
-            :_  this
-            =/  err
-              "no challenge to decline from {<who.action>}"
-            :~  [%give %poke-ack `~[leaf+err]]
-            ==
-          ::  tell our challenger we decline
-          :-  :~  :*  %pass  /poke/challenge/reply  %agent  [who.action %chess]
-                      %poke  %chess-decline-challenge  !>(~)
-                  ==
-              ==
-          %=  this
-            ::  remove our challenger from challenges-received
-            challenges-received  (~(del by challenges-received) who.action)
-          ==
-        %change-special-draw-preference
-          =/  game  (~(get by games) game-id.action)
-          ?~  game
-            :_  this
-            =/  err
-              "no active game with id {<game-id.action>}"
-            :~  [%give %poke-ack `~[leaf+err]]
-            ==
-          :-  :~  :*  %give
-                      %fact
-                      ~[/game/(scot %da game-id.action)/updates]
-                      %chess-update
-                      !>([%special-draw-preference game-id.action setting.action])
-                  ==
-              ==
-          %=  this
-            games  (~(put by games) game-id.action u.game(auto-claim-special-draws setting.action))
-          ==
+          =/  game-id
+            (mix now.bowl (end [3 6] eny.bowl))
+          :_  this
+          ::  attempt to accept game
+          ::  handle our end on ack
+          :~  :*  %pass
+                  /poke/game/(scot %da game-id)/init
+                  %agent
+                  [who.action %chess]
+                  %poke
+                  %chess-action
+                  !>([%game-accepted game-id our-side])
```

Comment:
Looks like there are some marks still to update throughout? I think `%chess-action` here should be `%chess-agent-action` and `%game-accepted` should be `%challenge-accepted`.